templates/virus_scan.sh

```diff
@@ -1,18 +1,28 @@
 #!/bin/bash
```

Comment:
This file should be renamed to `templates/virus_scan.tpl.sh` to match [our (undocumented) standard](https://github.com/cisagov/cool-assessment-terraform/tree/develop/cloud-init).